packages/faustwp-core/scripts/prebuild.js

```diff
@@ -0,0 +1,18 @@
+const fs = require('fs/promises');
+const path = require('path');
+
+function storeVersion() {
+  const version = require(path.join(
+    path.resolve(process.cwd()),
+    'package.json',
+  ))?.version ?? 'faust';
+  const content = `export const LIB_VERSION = '${version}';`;
```

Comment:
Should we rename `LIB_VERSION` -> `FAUST_VERSION` just to make things more clear?